code/jest.init.base.ts

```diff
@@ -58,41 +56,3 @@ const throwError = (message: any) => throwMessage('error: ', message);
 
 global.console.error = throwError;
 global.console.warn = throwWarning;
-
-// Mock for matchMedia since it's not yet implemented in JSDOM (https://jestjs.io/docs/en/manual-mocks#mocking-methods-which-are-not-implemented-in-jsdom)
-global.window.matchMedia = jest.fn().mockImplementation((query) => {
-  return {
-    matches: false,
-    media: query,
-    onchange: null,
-    addListener: jest.fn(), // deprecated
-    removeListener: jest.fn(), // deprecated
-    addEventListener: jest.fn(),
-    removeEventListener: jest.fn(),
-    dispatchEvent: jest.fn(),
-  };
-});
-class EventSourceMock {
-  static sources: EventSourceMock[] = [];
-
-  static reset() {
-    this.sources = [];
-  }
-
-  emitter: EventEmitter;
-
-  constructor() {
-    this.emitter = new EventEmitter();
-    EventSourceMock.sources.push(this);
-  }
-
-  addEventListener(event: string, cb: (data: any) => void) {
-    this.emitter.on(event, cb);
-  }
-
-  emit(event: string, data: any) {
-    this.emitter.emit(event, data);
-  }
-}
```

Comment:
❤️

Reply:
It all just moved into the browser version, but it means that the node tests won't have these mocks.

Reply:
right